### PR TITLE
PowerShell stub for deviceShifu 

### PR DESCRIPTION
**Pull request by tomqin93**
**What this PR does / why we need it**:
This PR adds PowerShell driver to deviceShifu

**Will this PR make the community happier**? 
Yes
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
- Adds PowerShell stub
- adds PowerShell specific commands to deviceShifu
```

### CODE REVIEW

**Comment by saiyan86** [comment]
Need to adjust CI pipeline in this PR too.

**Review by BtXin — DISMISSED** [review]

File: cmd/httpstub/powershellstub/powershellstub.go [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+func httpCmdlinePostHandler(resp http.ResponseWriter, req *http.Request) {
+	values := req.URL.Query()
+	for parameterName, parameterValues := range values {
+		klog.Infof("paramname is: %v, value is: %v", parameterName, parameterValues[0])
+	}
+
+	timeoutSeconds, err := strconv.Atoi(cmdExecTimeoutSecond)
+	if err != nil {
+		klog.Infof("cannot convert cmdExecTimeoutSecond: %v to integer", cmdExecTimeoutSecond)
```
Comment (kris21he) [inline cmd/httpstub/powershellstub/powershellstub.go:50]: klog.Errorf?

File: cmd/httpstub/powershellstub/powershellstub.go [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+	timeoutValue := req.URL.Query().Get("cmdTimeout")
+
+	if timeoutValue == "" {
+		klog.Infof("Url Param 'cmdTimeout' is missing, setting default to: %v", cmdExecTimeoutSecond)
+	} else {
+		timeoutSeconds, err = strconv.Atoi(timeoutValue)
+		klog.Infof("Setting timeout to: %v", timeoutSeconds)
+		if err != nil {
+			klog.Infof("cannot convert timeout param: %v to integer", timeoutValue)
```
Comment (kris21he) [inline cmd/httpstub/powershellstub/powershellstub.go:63]: klog.Errorf

File: cmd/httpstub/powershellstub/powershellstub.go [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+			resp.WriteHeader(http.StatusBadRequest)
+			return
+		}
+	}
+
+	httpCommand, err := ioutil.ReadAll(req.Body)
+	if err != nil {
+		panic(err)
```
Comment (kris21he) [inline cmd/httpstub/powershellstub/powershellstub.go:71]: do er need to log error here?

File: cmd/httpstub/powershellstub/powershellstub.go [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+	cmdString := string(httpCommand)
+	klog.Infof("running command: %v", cmdString)
+	posh := New()
+	stdOut, stdErr, err := posh.execute(timeoutSeconds, cmdString)
+	klog.Infof("ElevateProcessCmds:\nStdOut : '%s'\nStdErr: '%s'\nErr: %s", strings.TrimSpace(stdOut), stdErr, err)
+
+	if err != nil {
+		klog.Infof("Failed to run cmd: %v\n stderr: %v \n stdout: %v", cmdString, stdErr, stdOut)
+		resp.WriteHeader(http.StatusBadRequest)
```
Comment (kris21he) [inline cmd/httpstub/powershellstub/powershellstub.go:82]: internal server error?

**Review by kris21he — APPROVED** [review]
there still some place should print error log instead of info log.
besides that LGTM

**Pull request merged**